modules/tensor_mechanics/test/tests/critical_time_step/tests

```diff
@@ -0,0 +1,38 @@
+[Tests]
+design = 'CriticalTimeStep.md'
+issues = '#13975'
+  [./crit_time_solid_uniform]
+    type = 'CSVDiff'
+    input = 'crit_time_solid_uniform.i'
+    csvdiff = 'crit_time_solid_uniform_out.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    requirement = "The CriticalTimeStep class shall correctly compute the critical time step for a solid with uniform properties."
+    design = "CriticalTimeStep.md"
+  [../]
+  [./crit_time_solid_variable]
+    type = 'CSVDiff'
+    input = 'crit_time_solid_variable.i'
+    csvdiff = 'crit_time_solid_variable_out.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    requirement = "The CriticalTimeStep class shall correctly compute the critical time step for a solid with variable properties."
+    design = "CriticalTimeStep.md"
+  [../]
+  [./timoshenko_smallstrain_critstep]
+    type = 'CSVDiff'
+    input = 'timoshenko_smallstrain_critstep.i'
+    csvdiff = 'timoshenko_smallstrain_critstep_out.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-5
+    requirement = "The CriticalTimeStep class shall correctly compute the critical time step for a beam."
+    design = "CriticalTimeStep.md"
+  [../]
+  [./except1]
+    type = 'RunException'
+    input = 'non-isotropic_error_test.i'
+    expect_err = 'CriticalTimeStep can only be used with elasticity tensor materials that guarantee isotropic tensors.'
+    requirement = "The CriticalTimeStep class shall produce an error"
+                  " if the input elasticity tensor is non-isotropic."
```

Comment:
Requirements are pretty good! The only thing that our SQA consultant recommended is that you avoid specific object names because that's an implementation detail, not a requirement. Instead you should favor "The system shall contain a critical time step calculation that..."